desktop/apps/editorial_features/components/venice_2017/templates/index.jade

```diff
@@ -3,6 +3,7 @@ extends ../../../../../components/main_layout/templates/blank.jade
 append locals
   - assetPackage = 'editorial_features'
   - section = curation.get('sections')[videoIndex]
+  - options = {modal: false, stripe: false, sailthru: false, marketo: false}
```

Comment:
Sounds good, just returned sailthru to this template -- no, the how to view doesn't use this modal template.